etc/picongpu/summit-ornl/gpu_picongpu.profile.example

```diff
@@ -18,19 +18,19 @@ export proj=<yourProject>
 #export EDITOR="nano"
 
 # basic environment ###########################################################
-module load gcc/6.4.0
+module load gcc/8.1.1
 
 export CC=$(which gcc)
 export CXX=$(which g++)
 
 # required tools and libs
 module load git
-module load cmake/3.15.2
-module load cuda/10.1.168
+module load cmake/3.18.2
+module load cuda/10.1.243
 module load boost/1.66.0
 
 # plugins (optional) ##########################################################
-module load hdf5/1.10.3
+module load hdf5/1.10.4
 module load adios/1.13.1-py2 c-blosc zfp sz lz4
 module load ums
 module load ums-aph114
```

Comment:
loading this module causes the `zfp` module to change:
```
The following have been reloaded with a version change:
  1) zfp/0.5.2 => zfp/0.5.5
  ```
  Perhaps we should use `zfp/0.5.5` from the start. @psychocoderHPC what do you think?

Reply:
I did it first and running into an issue, I do not remember which. I can try again.

Reply:
I fixed it by loading `ums` first, this is what `spider` said to me.

Reply:
works perfectly :+1: